sw/device/lib/dif/dif_kmac.h

```diff
@@ -669,7 +675,9 @@ dif_result_t dif_kmac_end(const dif_kmac_t *kmac,
 
 /**
  * Read the kmac error register to get the error code indicated the interrupt
- * state.
+ * state. Note that if the ERR_CODE is a stale value that is left from previous
```

Comment:
For the stale value detection to work this assumes the INTR_STATE registerr will be cleared after any interrupt, and that interrupts were enabled.

Reply:
When I was updating the test, I had written some responses to your questions, as I updated the test. However, it turns out that I had left them on "pending" state so they were actually never visible to you. Sorry about this, and thanks for the review @matutem